docs/pages/guides/apple-privacy.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Privacy Manifests
+description: Learn about configuring iOS privacy manifests for your mobile app.
+---
+
+import { ConfigReactNative } from '~/components/plugins/ConfigSection';
+
+If you're using a native iOS library that uses a "restricted reason" APIs, you'll need to configure an iOS privacy manifest to declare why you're including native code calling those APIs.
+
+More details and a list of "required reason" APIs can be found in the [Apple Developer Documentation](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files).
+
+> **info** The information and steps included in this document are still in development and may change, either due to new tools built for this purpose or new requirements from Apple.
+
+## What is a Privacy Manifest?
+
+A privacy manifest is a file named **PrivacyInfo.xcprivacy** that is included in your iOS native project. This file is used to declare why the app includes native code that calls into certain APIs that Apple considers sensitive.
+
+Those APIs currently include accessing UserDefaults, file timestamp APIs, system boot time APIs, disk space APIs, and active keyboard APIs. Apple considers it an open list that can be expanded in the future.
+
+## Configuration in app.json/app.config.js
+
+You can include an iOS privacy manifest by using the `privacyManifests` field under `expo.ios` in your app config.
+
+### Example app.json with `privacyManifests` config
+
+```json
+{
+  "expo": {
+    "name": "My App",
+    "slug": "my-app",
+    // other fields...
+    "ios": {
+      "privacyManifests": {
+        "NSPrivacyAccessedAPITypes": [
+          {
+            "NSPrivacyAccessedAPIType": "NSPrivacyAccessedAPICategoryUserDefaults",
+            "NSPrivacyAccessedAPITypeReasons": ["CA92.1"]
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+<ConfigReactNative>
+
+You can include an iOS privacy manifest in a bare Expo app by creating a **PrivacyInfo.xcprivacy** file using Xcode and adding it to your iOS app target.
+Follow [this guide](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files) to create a **PrivacyInfo.xcprivacy** file.
```

Comment:
```suggestion
Follow [Apple's Privacy manifest files](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files) guide to create a **PrivacyInfo.xcprivacy** file.
```

- Let's be explicit about the [link as per writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#linking-to-other-docs)